### ISSUE TITLE
Accessiblity: Header tags on landing page need to start with (h1) and remove one empty (h?) tag.

### ISSUE THREAD

**Issue by Dawnzar** [CityofOttawa/ckanext-ottawa#1]
WCAG 2.0 - Success Criteria - 1.3.1 Level (A) violation
1.Go to: http://data.ottawa.ca/
2.View source
3.'H1' tag is missing as it starts with 'H2'
4. Also there is an 'empty' heading second last from the bottom of the page.

Resolution: Update tags to begin with 'H1" & remove the empty one (h?) second from the bottom of the page.
![main pg_h list](https://cloud.githubusercontent.com/assets/5271253/8012321/961dc7ea-0b8e-11e5-9e28-254ed4bb902f.JPG)

**Comment by Dawnzar** [comment]
Will use this one.

**Comment by deniszgonjanin** [comment]
@Dawnzar actually this repo is the correct one. The other one - https://github.com/CityofOttawa/Ottawa-ckan - is the old site that has been replaced by this new one.

No worries though, we can work off the issues from the other repository, but you could open any new issues on this one.

**Comment by Dawnzar** [comment]
I have moved them over…better to have them all in the right area. Thanks for letting me know.

**Comment by deniszgonjanin** [comment]
Great, I'll put a note on the other repo that it's only an archive of an old project
